### ISSUE TITLE
Create Dashboard / front page

### ISSUE THREAD

**Issue by AriHrannar** [veritus/veritus-web#58]
# Description
1. Rank politician by how many promises they have delivered on
2. Rank parties on how many promises they have delivered on

**Comment by AriHrannar** [comment]
@Ragnar-H @skabbi any ideas on what should appear on the Dashboard / front page? Im thinking some kind of statistics that might be interesting at first glance

**Comment by skabbi** [comment]
Having statistics is interesting, what statistics are we able to show? 
Though I worry that if the first thing the user sees is favorable/unfavorable statistic it might accidentally give the impression the site is biased.

**Comment by AriHrannar** [comment]
Yeah it has to be 100% fact based, purely on numbers we have in the system. 

My idea is to have a place where users can drop in and see what politicians are doing well and which ones are doing badly. How to represent "well" and "badly" in an unbiased statistical way is the debate! I suspect this might be a bit trial and error.

The data we have now is promises, cases, subjects (what we use to categorize the promises and cases and then find relations between them) and voting records on those cases.

**Comment by Ragnar-H** [comment]
>I suspect this might be a bit trial and error.

💯 agree with this.

I'm also on the boat of "show how many promises, politicians and parliaments are in the system".
As a sort of "this is what we are basing our ratings on"

**Comment by Ragnar-H** [comment]
Dunno how much help that 👆 comment is.
Just a random coffee-thought :)
